doc/sphinx/source/api/index.rst

```diff
@@ -12,22 +12,24 @@ Public API
 These objects and functions are intended to be used by general users of libCEED
 and can generally be found in `ceed.h`.
 
-..
-   mermaid::
-
-   graph TD
-   U -->|wrap data| V
-   U(High-level user code) -->|apply| O
-   subgraph Ceed
-   O(CeedOperator) --> E(CeedElemRestriction)
-   O --> B(CeedBasis)
-   O --> Q(CeedQFunction)
-   B --> V(CeedVector)
-   O --> V
-   E --> V
-   Q --> V
-   end
-   Q --> UQ(User's physics)
+
+.. only:: html
```

Comment:
@jedbrown I have enabled this mermaid graph for the html build only. Was this the desired result that you were hoping to get in [c5410](https://github.com/CEED/libCEED/pull/446/commits/c5410b0bd1538923eb856f2f724e1544075c923d)

Reply:
I see that the graph is a bit jumbled in the html version, but at least the latexpdf builder does not break this way.